### PR TITLE
CI: travis: add IMX8M build check

### PR DESCRIPTION
**Pull request by dbaluta**
Enable imx8m build check in travis.

Signed-off-by: Daniel Baluta <daniel.baluta@nxp.com>

### CODE REVIEW

**Comment by xiulipan** [comment]
@dbaluta Could you merge this change with https://github.com/thesofproject/sof/pull/2433 so that we can have a per test inside a PR

File: .travis.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ env:
   - PLATFORM=jsl
   - PLATFORM=imx8
   - PLATFORM=imx8x
+  - PLATFORM=imx8x
```
Comment (mihaix) [inline .travis.yml]: imx8m?

Comment (dbaluta) [reply .travis.yml]: @mihaix thanks for noticing. will fix

Comment (dbaluta) [reply .travis.yml]: @mihaix fixed thanks!

**Comment by dbaluta** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
CI is either licence timeouts or existing issue that needs bisect.

**Comment by dbaluta** [comment]
@lgirdwood noticed that there is problem with license on HSW with xtensa-cc

**Pull request merged**